deploy/cr.yaml

```diff
@@ -427,6 +427,7 @@ spec:
 #    security:
 #      encryptionKeySecret: "my-cluster-name-mongodb-encryption-key"
 
+  # More info can be found at https://docs.percona.com/percona-backup-mongodb/index.html
   backup:
```

Comment:
@vsychov could you please check @spron-in comments? 
@vorbidan we try to keep our CR up-to-date.  Did you meet any specific problems with it? What kind of changes except these comments would be useful for you?

Reply:
@nmarukovich - My comment was regarding the proposal by @vsychov to add comments in cr.yaml to describe properties and values. IMO, the CR Definition as an OpenAPI is the right place for this sort of content. 
Even though it looks tempting to add multiline comments int CR example, it seems to be counter to the goal - making it more clear.
Having a pointer to docs is OK. Maybe also a pointer to an OpenAPI for the type description?